File: sem 5/PHP/UNIT_2/Qu2.php
<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title></title>
</head>
<body>
	<form method="post">
		Enter the age:<input type="number" name="age">
		<br>
		<input type="submit" name="Submit">
	</form>
</body>
</html>

<?php
	if(isset($_POST['age']))
	{
		$age=$_POST['age'];

		if($age>=18)
		{
			echo "<h1>Eligible to Vote</h1>";
		}
		else
		{
			echo "<h1>Not Eligible</h1>";
		}
	}
?>

File: sem 5/Python/LAB TASK UNIT-2/Qu2.py
r=int(input("Enter the number of rows : "))

for i in range(1,r+1):
    for space in range(r-i):
        print(" ",end="")

    for j in range(1,2*i):
        if j==1 or j==2*i-1 or i==r:
            print("*",end="")
        else:
            print(" ",end="")
    print()